internal/system/system.go

```diff
@@ -425,3 +425,94 @@ func CopyFile(src string, dest string) error {
     return nil
 }
 
+func CheckUpdates() {
+	fmt.Println("Checking for Edgebox System Updates.")
+	
+	// Configure the service and start it
+	cmd := exec.Command("sh", "/home/system/components/updater/run.sh", "--check")
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		panic(err)
+	}
+	scanner := bufio.NewScanner(stdout)
+	err = cmd.Start()
+	if err != nil {
+		panic(err)
+	}
+	for scanner.Scan() {
+		// fmt.Println(scanner.Text())
+		text := scanner.Text()
+		fmt.Println(text)
+	}
```

Comment:
Needs to handle not being able to run the updater. We don't want to crash in that case but gracefully exit.
Would be maybe useful to use the `return_val, err :=` pattern commonly found in go built-in methods to return a proper error back to whatever runs CheckUpdates(). This will also allow us to mark these tasks as errors (and that thus reflected in the dashboard / api)